applications/postgres.md

```diff
@@ -1,13 +1,215 @@
+
+``` 
+
+ Now let's deploy PostgreSQL using following commands:
+
+`kubectl apply -f px-postgres-app.yaml`
+
+### Validate StorageClass, PersistentVolumeClaim and PostgreSQL Deployment.
+```
+ubuntu@node1:~$ kubectl get sc
+NAME             PROVISIONER                     AGE
+px-postgres-sc   kubernetes.io/portworx-volume   1h
+ubuntu@node1:~$ kubectl get pvc
+NAME            STATUS    VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS     AGE
+postgres-data   Bound     pvc-60e43292-06e3-11e8-96b4-022185d04910   1Gi        RWO            px-postgres-sc   1h
+ubuntu@node1:~$ kubectl get pod
+NAME                        READY     STATUS    RESTARTS   AGE
+postgres-86cb8587c4-l9r48   1/1       Running   0          1h
+```
+
+ 
+### Access PostgreSQL DB Server and Validate 
+ 
+To access via docker exec:
+
+```
+ubuntu@node2:~$ docker ps -a | grep postgres
+k8s_pgbench_postgres-86cb8587c4-l9r48_default_7a8af36e-06e3-11e8-96b4-022185d04910_0
+e7bb6aa3586f        postgres@sha256:2f4c2e4db86a1762de96a2331eb4791f91b6651d923792d66d0f4d53c8d67eed                                                         "docker-entrypoint..."   57 minutes ago      Up 57 minutes                                 k8s_postgres_postgres-86cb8587c4-l9r48_default_7a8af36e-06e3-11e8-96b4-022185d04910_0
+f44e191530c7        gcr.io/google_containers/pause-amd64:3.0                                                                                                 "/pause"                 58 minutes ago      Up 58 minutes                                 k8s_POD_postgres-86cb8587c4-l9r48_default_7a8af36e-06e3-11e8-96b4-022185d04910_0
+ubuntu@node2:~$ docker exec -it e7bb6aa3586f bin/bash
+root@postgres-86cb8587c4-l9r48:/#
+root@postgres-86cb8587c4-l9r48:/# psql
+psql (9.5.10)
+Type "help" for help.
+
+pgbench=# \d
+              List of relations
+ Schema |       Name       | Type  |  Owner
+--------+------------------+-------+---------
+ public | pgbench_accounts | table | pgbench
+ public | pgbench_branches | table | pgbench
+ public | pgbench_history  | table | pgbench
+ public | pgbench_tellers  | table | pgbench
+(4 rows)
+
+pgbench=# \l
+                                 List of databases
+   Name    |  Owner   | Encoding |  Collate   |   Ctype    |   Access privileges
+-----------+----------+----------+------------+------------+-----------------------
+ pgbench   | postgres | UTF8     | en_US.utf8 | en_US.utf8 |
+ postgres  | postgres | UTF8     | en_US.utf8 | en_US.utf8 |
+ template0 | postgres | UTF8     | en_US.utf8 | en_US.utf8 | =c/postgres          +
+           |          |          |            |            | postgres=CTc/postgres
+ template1 | postgres | UTF8     | en_US.utf8 | en_US.utf8 | =c/postgres          +
+           |          |          |            |            | postgres=CTc/postgres
+(4 rows)
+
+pgbench=# \q
+root@postgres-86cb8587c4-l9r48:/# exit
+
+```
+
+### Database recoverability with Portworx snapshots
```

Comment:
For Kubernetes snapshots, we should use https://docs.portworx.com/scheduler/kubernetes/snaps.html